docs/docs/training-data-format.mdx

```diff
@@ -458,6 +494,37 @@ stories:
   - action: action_store_feedback
 ```
 
+If you want to put bot utterance directly in the stories, use `bot:` key followed by
+the text that you want your bot to say.
```

Comment:
Similarly, I have no idea how this works. Does ted just make predictions ranking "utter_greet" against "How are you?" in a greet bot and go from there? If I have some stories where I use "utter_howareyou" and others where I provide the utterance directly, is that going to confuse TED?

Reply:
added explanations